tests/Kerbero.Infrastructure.Tests/NukiActions/Repositories/NukiSmartLockExternalRepositoryTests.cs

```diff
@@ -29,6 +31,10 @@ public NukiSmartLockExternalRepositoryTests()
             BaseUrl = "http://api.nuki.io"
         }), helper);
         _httpTest = new HttpTest();
+        
+        using var streamReader = new StreamReader("JsonData/get-nuki-smartlock-response.json");
```

Comment:
```suggestion
        using var streamReader = new StreamReader("JsonData/get-nuki-smartlock-response.json");
```
why not File sync methods?